### ISSUE TITLE
regression in 4.1.1: ${target-base} now an unknown manifest token

### ISSUE THREAD

**Issue by drnic** [cloudfoundry-attic/cf#23]
This manifest below was created by cf recently, but the latest cf gem 4.1.1 now won't consume it, giving an error `RuntimeError: Unknown symbol in manifest: target-base`

```

---
applications:
- name: name
  memory: 256M
  instances: 1
  url: name.${target-base}
  path: .
```

**Comment by vito** [comment]
IMO there should be a step added to the normalizer that translates this symbol to a new more generic 'app-domain' symbol or the like, which would resolve to the shared domain of the deployment.

Commit that removed target-base: https://github.com/cloudfoundry/cf/commit/dd13b956360deb69315ef42e818cde2ec7d2fd7b

Normalizer: https://github.com/cloudfoundry/cf/blob/master/lib/manifests/loader/normalizer.rb

The new variable would resolve to something like `client.current_space.domains.find { |d| d.owning_organization.nil? }.name`.

**Comment by cf-frontend** [comment]
This was a mistake. We no longer generate manifest files that include target-base, but old ones should continue to work. This should be fixed as of 4.1.3
